app/views/catalog/_print.html.erb

```diff
@@ -0,0 +1,23 @@
+<%# file _print.html.erb %>
+<% @documents.entry.each do |document| %>
+
+  <h3 class="modal-title"><%= document_heading(document) %></h3>
+
+  <%# PUT document.send(:print) here; example from citation below %>
+
+       <% if document.respond_to?(:export_as_mla_citation_txt) %>
+         <h4><%#= t('blacklight.citation.mla') %></h4>
+         <%3= document.send(:export_as_mla_citation_txt).html_safe %><br/><br/>
```

Comment:
What does the '3' here do? (or is it just a typo?)

Reply:
Yes, it is a typo!